cadquery/assembly.py

```diff
@@ -197,6 +197,28 @@ def add(
         """
         ...
 
+    @overload
```

Comment:
Any reason for adding additional overload and not just updating?

Reply:
I figured it was safer to leave the existing overloads alone. I can combine it with the one above if you want.

Reply:
OK, I removed it.